doc/classes/HTTPRequest.xml

```diff
@@ -80,7 +80,7 @@
 		}
 		[/csharp]
 		[/codeblocks]
-		[b]Example of loading and displaying an image using HTTPRequest:[/b]
+		[b]Example:[/b] Load and display an image using [HTTPRequest]:
```

Comment:
Reading this makes me cringe a bit (not the edit, the original too), as HTTPRequest has nothing to do with displaying an image. So "Example: Load an image using [HTTPRequest] and display it" would be more correct, but more cumbersome, so... whatever, let's probably leave as is.

Reply:
No, that seems appropriate to me. I can change it.